alf/optimizers/traj_optimizers.py

```diff
@@ -50,12 +54,15 @@ def __init__(self,
                  lower_bound=None):
         """Random Trajectory Optimizer
 
+        This module conducts trajectory optimization via random-shooting-based
+            optimization.
```

Comment:
To be more concrete such as: generating a random population for each sample in the batch and select those having the lowest cost.

Reply:
Added concrete description.